webhelper/src/gwh-plugin.c

```diff
@@ -419,11 +419,14 @@ plugin_init (GeanyData *data)
    * (g_quark_from_static_string() for example) so it's not safe to remove it */
   plugin_module_make_resident (geany_plugin);
   
-  /* webkit uses threads but don't initialize the thread system */
+  /* Deprecated function g_thread_init() is not needed. after glib 
+   * versions >= 2.32 
+  
   if (! g_thread_supported ()) {
     g_thread_init (NULL);
   }
   
+   */
```

Comment:
don't comment out stuff, just keep it or remove it.  we use a VCS anyway, it can be recovered if it really need to